core/field_vertical_separator.js

```diff
@@ -22,140 +22,133 @@
+      return;
+    }
+    const height = 10 * this.getConstants().GRID_UNIT;
+    this.size_ = new Blockly.utils.Size(1, height);
+
+    // Build the DOM.
+    /** @type {SVGElement} */
+    this.fieldGroup_ = Blockly.utils.dom.createSvgElement('g', {}, null);
+    if (!this.visible_) {
+      this.fieldGroup_.style.display = 'none';
+    }
+    /** @type {SVGElement} */
+    this.lineElement_ = Blockly.utils.dom.createSvgElement('line',
+        {
+          'stroke': this.sourceBlock_.getColourSecondary(),
+          'stroke-linecap': 'round',
+          'x1': 0,
+          'y1': 0,
+          'x2': 0,
+          'y2': height
+        }, this.fieldGroup_);
+
+    this.sourceBlock_.getSvgRoot().appendChild(this.fieldGroup_);
+  }
 
-/**
- * Dispose of all DOM objects belonging to this text.
- */
-Blockly.FieldVerticalSeparator.prototype.dispose = function() {
-  goog.dom.removeNode(this.fieldGroup_);
-  this.fieldGroup_ = null;
-  this.lineElement_ = null;
-};
+  /**
+   * Set the height of the line element, without adjusting the field's height.
+   * This allows the line's height to be changed without causing it to be
+   * centered with the new height (needed for correct rendering of hat blocks).
+   * @param {number} newHeight the new height for the line.
+   * @package
+   */
+  setLineHeight(newHeight) {
+    this.lineElement_.setAttribute('y2', newHeight);
+  };
+
+  /**
+   * Dispose of all DOM objects belonging to this text.
+   */
+  dispose() {
+    this.fieldGroup_.remove();
+    this.fieldGroup_ = null;
+    this.lineElement_ = null;
+  }
 
-/**
- * Get the value of this field. A no-op in this case.
- * @return {string} null.
- * @override
- */
-Blockly.FieldVerticalSeparator.prototype.getValue = function() {
-  return null;
-};
+  /**
+   * Get the value of this field. A no-op in this case.
+   * @return {string} null.
+   * @override
+   */
+  getValue() {
+    return null;
+  }
 
-/**
- * Set the value of this field. A no-op in this case.
- * @param {?string} src New value.
- * @override
- */
-Blockly.FieldVerticalSeparator.prototype.setValue = function(
-    /* eslint-disable no-unused-vars */ src
-    /* eslint-enable no-unused-vars */) {
-  return;
-};
+  /**
+   * Set the value of this field. A no-op in this case.
+   * @param {?string} src New value.
+   * @override
+   */
+  setValue(
+      /* eslint-disable no-unused-vars */ src
+      /* eslint-enable no-unused-vars */) {
+    return;
+  }
 
-/**
- * Set the text of this field. A no-op in this case.
- * @param {?string} alt New text.
- * @override
- */
-Blockly.FieldVerticalSeparator.prototype.setText = function(
-    /* eslint-disable no-unused-vars */ alt
-    /* eslint-enable no-unused-vars */) {
-  return;
-};
+  /**
+   * Set the text of this field. A no-op in this case.
+   * @param {?string} alt New text.
+   * @override
+   */
+  setText(
+      /* eslint-disable no-unused-vars */ alt
+      /* eslint-enable no-unused-vars */) {
+    return;
+  }
```

Comment:
This function doesn't exist anymore so you should be able to delete it.

Reply:
Done